assets/bib/pubmed_krauthammer.bib

```diff
@@ -1,3 +1,19 @@
+@Article{Allam2019,
```

Comment:
`Article` needs to be lowercase in order for the entry to be recognized by bibbase. When I run this locally, the article does not display.

Reply:
It shows up on https://uzh-dqbm-cmi.github.io/cmi-website/publications/

Reply:
Oh, running it locally doesn't actually use the local copy of the .bib file, so I bet I just checked it before the hosted file got updated in github. Never mind! 